patterns/README.md

```diff
@@ -25,7 +25,7 @@ These patterns deal with object creation mechanisms, trying to create objects in
 | [Abstract Factory](./abstract-factory/) | Provides an interface for creating families of related or dependent objects without specifying their concrete classes.|
 | Builder         | Separates the construction of a complex object from its representation, allowing the same construction process to create various representations.|
 | Factory Method  | Defines an interface for creating an object but lets subclasses alter the type of objects that will be created.     |
-| Prototype       | Creates new objects by copying an existing object, known as the prototype.  
+| [Prototype](./Prototype/)       | Creates new objects by copying an existing object, known as the prototype.  
```

Comment:
I think you should move this into a different folder: patterns\Creational\Prototype